README.md

```diff
@@ -36,7 +36,35 @@ class MyWidget extends StatelessWidget {
 
 View the Flutter app in the `example` directory to see all the available `FontAwesomeIcons`.
 
+## Pro icons
+
+If you bought Font Awesome Pro icons, you can import them here. 
+
+:exclamation: By importing Pro icons you acknowledge that it is your obligation to keep these files private. This includes **not** uploading your package to GitHub or other public file sharing services.
+
+- Download and add the light TTF to your project (for example, under `fonts/fa-light-300.ttf`)
```

Comment:
Yes, I've noted in the OP that this doesn't include all icons, and that we would need to regenerate the list with a Pro `icons.json`

Reply:
- As the list for pro icons is distributed with the paid version of font awesome, I'm pretty sure we cannot add them here, but the developer who bought them has to do this.
- Adding the fa-light font will not add regular and solid pro icons, their files would need to be swapped as well - which is not possible without rebuilding the package like in the original proposal

Reply:
We would need to check with Font Awesome if they would allow to use the Pro `icons.json` (which is just a list at the end of the day, with the same info as their website). I do believe they would allow it due to this helping them sell more

Reply:
One of the two situations would arise:
1. We would have icon definitons which do not work for non-pro users -> frustrating
2. We would need another class (like `FontAwesomeIconsPro`), which then would require the developer to remember if an icon is in the pro package or not -> frustrating

So I really don't see a benefit. I agree with @brianegan, the api should be kept as clean as possible.
Apart from that, if a developer uses the pro icons, he has access to the fonts and the `icons.json`. The fonts need to be added by hand, so is it really that much of a problem to run a little tool as well?

Also, again, this is not only about the `icons.json`, the pro `fa-solid` and `fa-regular` `ttf`s need to be swapped as well, which cannot be done without rebuilding the package as suggested in the original proposal. You cannot just [add the pro fonts as normal fonts](https://github.com/brianegan/font_awesome_flutter/pull/37#discussion_r294029653).